src/common/low_precision_transformations/src/layer_transformation.cpp

```diff
@@ -442,6 +446,16 @@ void LayerTransformation::addPattern(ngraph::pass::GraphRewrite& pass, Transform
     NGRAPH_SUPPRESS_DEPRECATED_END
 }
 
+void LayerTransformation::setDefaultPrecisions(const std::vector<ngraph::element::Type>& precisions) {
+    std::lock_guard<std::mutex> lock(defaultPrecisionsMutex);
+    defaultPrecisions = precisions;
```

Comment:
how this solved the issue? 
If CPU set one set of precisions and runs LPT pipeline, GPU sets different set of precisions while CPU LPT pipeline is running, then CPU LPT pipeline can be affected anyway